packages/angular_devkit/build_angular/src/browser/tests/options/main_spec.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { buildWebpackBrowser } from '../../index';
+import { BASE_OPTIONS, BROWSER_BUILDER_INFO, describeBuilder } from '../setup';
+
+describeBuilder(buildWebpackBrowser, BROWSER_BUILDER_INFO, (harness) => {
+  beforeAll(() => {
+    harness.useProject('test', { root: '.', sourceRoot: 'src' });
```

Comment:
I think in most cases the the `harness.useProject('test', { root: '.', sourceRoot: 'src' });` will be the same for the majority of tests.

This is because, if either one of these options changes, changes need to be done in the underlying test project.

Hence, wouldn't be better that the `useProject` is not needed, or if needed it has default values?

Reply:
Good point.  I’ll set those as the default.